Makefile

```diff
@@ -0,0 +1,20 @@
+DOCKER_BUILD_IMAGE ?= assisted-grafana:latest
+
+.PHONY: all
+all: validate
+
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+.PHONY: docker-build
+docker-build: ## Build docker image to run Makefile targets
+	docker build -t $(DOCKER_BUILD_IMAGE) -f Dockerfile.build .
```

Comment:
is it possible to support podman as well?

Reply:
If not, at least renaming things that have ``docker`` in them, and replace with a more meaningful name to their task

Reply:
I think we need to agree as a team what tool we want to support, I really think that supporting both it's a waste of time, as they diverge in advance features.

The idea was to have the same target to run within the build container (in this case with docker, but we can discuss if podman would suit better), and without container (within the same build container, but passed as an image instead of a command)

Reply:
As discussed in other channels with @osherdp , we will try to see if it's feasible to only support podman going forward.

In this PR's context, I'll change all commands/dockerfile to support only podman.